ts/Series/XRangeSeries.ts

```diff
@@ -696,7 +696,7 @@ BaseSeries.seriesType<typeof Highcharts.XRangeSeries>('xrange', 'column'
                         }
                         if (isObject(seriesOpts.partialFill)) {
                             pfOptions = merge(
-                                pfOptions, seriesOpts.partialFill
+                                {}, seriesOpts.partialFill, pfOptions
```

Comment:
The first empty object is not needed, since merge (by default) returns a new object and assigns it to `pfOptions`.